internal/benchrunner/runners/system/runner.go

```diff
@@ -362,6 +362,11 @@ func (r *runner) createBenchmarkPolicy(pkgManifest *packages.PackageManifest) (*
 		MonitoringEnabled: []string{"logs", "metrics"},
 	}
 
+	// Assign the data_output_id to the agent policy to configure the output to logstash. The value is inferred from stack/_static/kibana.yml.tmpl
+	if r.options.Profile.Config("stack.logstash_enabled", "false") == "true" {
+		p.DataOutputID = "fleet-logstash-output"
```

Comment:
Could we add a test for this case?

Reply:
@jsoriano You mean a test package?

Reply:
It would be a test that runs benchmarks with logstash, it can be a new package or a new script and an existing package, not sure what would be the best approach.

Reply:
I can add a regression test. As currently logstash does not provide relevant stats that are comparable to ingest node stats. But I just want to make we can run system benchmarks even with logstash enabled.

Reply:
@jsoriano Added a test package